core/templates/AppSpec.ts

```diff
@@ -29,6 +29,7 @@ import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
 import constants from 'assets/constants';
+// Import from assets/constants because we loop over all constants.
```

Comment:
I do not understand this comment.

Reply:
maybe we don't need this comment, I will delete it.

Reply:
done